template/Vagrantfile.erb

```diff
@@ -37,11 +37,23 @@ Vagrant.configure('2') do |config|
 
   # config.vm.network :forwarded_port, :host => 9200, :guest => 9200
 
+  $script = %q{
```

Comment:
I was simply following an example on the vagrant website :/  If folks feel strongly I will change it.